### PR TITLE
Distributed Training scripts for PaddleClas Benchmark. 

### PR DESCRIPTION
**Pull request by GhostScreaming**
Add distributed training scripts for ResNet50_bs128_fp32, ShuffleNetV2_x1_0_bs1536_fp32, MobileNetV3_large_x1_0_bs640_fp32 and SwinTransformer_tiny_patch4_window7_224_bs104_fp32. Now these benchmark can be tested on multi nodes. And fix bug of HRNet. When multi-process create the same folder, it may cause conflict.

### CODE REVIEW

File: frame_benchmark/pytorch/dynamic/PaddleClas/scripts/HRNet_W48_C/benchmark_common/utils.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+import torch
+import torch.optim as optim
+
+def create_logger(cfg, cfg_name, phase='train'):
+    root_output_dir = Path(cfg.OUTPUT_DIR)
+    # set up logger
+    if not root_output_dir.exists():
+        print('=> creating {}'.format(root_output_dir))
+        root_output_dir.mkdir()
```
Comment (sneaxiy) [inline frame_benchmark/pytorch/dynamic/PaddleClas/scripts/HRNet_W48_C/benchmark_common/utils.py:25]: Add `exist_ok=True`.

File: frame_benchmark/pytorch/dynamic/PaddleClas/scripts/MobileNetV3_large_x1_0/benchmark_common/run_benchmark.sh [hunk 2]
```diff
@@ -29,8 +29,16 @@ function _set_params(){
...
+    multi_nodes=${MULTINODES:-"false"}      # (必选) 多卡训练开关，默认关闭，通过全局变量传递
+    node_num=${9:-"2"}                      #（可选） 节点数量
+    node_rank=${10:-"0"}                    # (可选)  节点rank
+    master_addr=${11:-"127.0.0.1"}       # (可选) 主节点ip地址
+    master_port=${12:-"1928"}               # (可选) 主节点端口号
+    # Added for distributed training
+
     #   以下为通用拼接log路径，无特殊可不用修改
-    model_name=${model_item}_bs${base_batch_size}_${fp_item}_${run_process_type}_${run_mode}  # (必填) 切格式不要改动,与平台页面展示对齐
+    model_name=${model_item}_bs${base_batch_size}_${fp_item}_${run_mode}  # (必填) 切格式不要改动,与平台页面展示对齐
```
Comment (sneaxiy) [inline frame_benchmark/pytorch/dynamic/PaddleClas/scripts/MobileNetV3_large_x1_0/benchmark_common/run_benchmark.sh:41]: Please let Lixin conform whether this change is OK.

**Review by mmglove — APPROVED** [review]
